src/features/SelectGroup/SelectGroup.scss

```diff
@@ -0,0 +1,11 @@
+.selectGroup {
+  & .react-select__control,
```

Comment:
Please remove the parent selector `&`. Because if you remove it nothing will change. When you put these classes `.react-select__…` into `.selectGroup` their specificities already increased up to 2 classes